docs/authelia-from-scratch.md

```diff
@@ -0,0 +1,448 @@
+# Installing **Authelia** on bare metal from scratch
```

Comment:
Please specify it's for debian in the title.

Reply:
Done :)